### PR TITLE
[SASS-3296] Create -"Did your pension schemes pay tax on the amount that resulted in a transfer charge?"

### PR DESCRIPTION
**Pull request by adam-shahzad-2**
# Checklist

Jira subtask: SASS-3296

## Developer: Before Asking Others To Review Your Code

Have you ...

- [x]  Had a QA verify that all acceptance criteria have been met, with comment(s) in the ticket? 
- [x]  Hosted a group review of your implementation?
- [x]  Rebased against `main`? 
- [x]  Ensured that your commit(s) contain the ticket key?
- [x]  Ensured that your branch is named after the ticket key?
- [x]  Successfully run the unit tests?
- [x]  Successfully run the integration tests?
- [ ]  Successfully run the journey tests? If not, please add a comment explaining why it wasn't necessary.
- [x]  Ensured that the PR job has passed.

## Reviewer: Before Reviewing

Have you ...

- [x]  Pulled the branch down?
- [x]  Assigned yourself to this PR?
- [ ]  Moved the Jira *subtask* to “in review”?

## Developer: After Approval

Have you ...

- [ ]  Moved the Jira *subtask* to “in pipeline”?

## Developer: After Merging

Have you ...

- [ ]  Scheduled a demo? If not, please add a comment.

### CODE REVIEW

File: test/views/OverseasPensionTransferTaxChargeSpec.scala [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+      }
+
+      "render the page with an error when the user doesn’t select a radio button" which {
+        val agentOrIndividual = if (userScenario.isAgent) "agent" else "individual"
+        implicit val messages: Messages = getMessages(userScenario.isWelsh)
+        implicit val userSessionDataRequest: UserSessionDataRequest[AnyContent] = new UserSessionDataRequest(aPensionsUserData, aUser, if (userScenario.isAgent) fakeAgentRequest else fakeIndividualRequest)
+        implicit val htmlFormat = underTest(form(agentOrIndividual).bind(Map(RadioButtonAmountForm.yesNo -> "")), taxYearEOY)
+        implicit val document: Document = Jsoup.parse(htmlFormat.body)
+
+        errorSummaryCheck(userScenario.specificExpectedResults.get.expectedNoEntryErrorText, "#value")
```
Comment (salman-a-dev) [inline test/views/OverseasPensionTransferTaxChargeSpec.scala:165]: You may also want to check "errorAboveElementCheck" for all of the cases where relevant.

File: app/controllers/pensions/transferIntoOverseasPensions/PensionSchemeTaxTransferController.scala [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+import utils.{Clock, SessionHelper}
+import views.html.pensions.transferIntoOverseasPensions.pensionSchemeTaxTransferChargeVeiw
+
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+class PensionSchemeTaxTransferController @Inject()(
+                                                    actionsProvider: ActionsProvider,
+                                                    pensionSessionService: PensionSessionService,
+                                                    view: pensionSchemeTaxTransferChargeVeiw,
```
Comment (salman-a-dev) [inline app/controllers/pensions/transferIntoOverseasPensions/PensionSchemeTaxTransferController.scala:37]: view is misspelled "pensionSchemeTaxTransferCharge**Veiw**"

**Comment by salman-a-dev** [comment]
Is [app/views/pensions/transferIntoOverseasPensions/transferChargeVeiw.html] supposed to be empty?

File: app/controllers/pensions/transferIntoOverseasPensions/PensionSchemeTaxTransferController.scala [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import services.PensionSessionService
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
+import utils.{Clock, SessionHelper}
+import views.html.pensions.transferIntoOverseasPensions.pensionSchemeTaxTransferChargeVeiw
+
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+class PensionSchemeTaxTransferController @Inject()(
```
Comment (salman-a-dev) [inline app/controllers/pensions/transferIntoOverseasPensions/PensionSchemeTaxTransferController.scala:34]: add a @singleton annotation

**Comment by adam-shahzad-2** [comment]
> Is [app/views/pensions/transferIntoOverseasPensions/transferChargeVeiw.html] supposed to be empty?

removed

**Review by salman-a-dev — APPROVED** [review]

**Pull request merged**